src/status_im2/contexts/chat/messages/list/view.cljs

```diff
@@ -318,6 +324,15 @@
        :render-fn                         render-fn
        :on-viewable-items-changed         on-viewable-items-changed
        :on-content-size-change            (fn [_ y]
+                                            (if (or
+                                                 (and keyboard-shown?
+                                                      less-than-two-messages?
+                                                      (> 100 (reanimated/get-shared-value scroll-y))
+                                                      (< 560
+                                                         (reanimated/get-shared-value content-height)))
+                                                 (< 135 (reanimated/get-shared-value scroll-y)))
```

Comment:
it's all full of magic numbers 🥲
could you at least move them to the to constants above to make them less random?